### ISSUE TITLE
`default-features = false` does not build

### ISSUE THREAD

**Issue by girlbossceo** [Canop/termimad#62]
Was trying to use `default-features = false` in one of my projects for termimad and I seem to get this error:

```
error[E0609]: no field `composite` on type `&composite::FmtComposite<'_>`
   --> /Users/strawberry/.cargo/registry/src/index.crates.io-6f17d22bba15001f/termimad-0.29.3/src/skin.rs:565:22
    |
565 |         for c in &fc.composite.compounds {
    |                      ^^^^^^^^^ unknown field
    |
    = note: available fields are: `kind`, `compounds`, `visible_length`, `spacing`

For more information about this error, try `rustc --explain E0609`.
error: could not compile `termimad` (lib) due to 1 previous error
```

**Comment by Canop** [comment]
Indeed.
I'll fix this.

**Comment by Canop** [comment]
If you manage to check compiling your project with the PR's branch, it's better. If not, I'll merge it tomorrow.

**Comment by girlbossceo** [comment]
Looks like that fixed it, thank you!

**Comment by Canop** [comment]
Released version 0.29.4
